test/e2e/clusterlogging_test.go

```diff
@@ -13,13 +13,13 @@ import (
 
 	logging "github.com/openshift/cluster-logging-operator/pkg/apis/logging/v1alpha1"
 	framework "github.com/operator-framework/operator-sdk/pkg/test"
-	rbac "k8s.io/api/rbac/v1beta1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	rbac "k8s.io/api/rbac/v1beta1"
 )
 
 var (
 	retryInterval        = time.Second * 5
-	timeout              = time.Second * 60
+	timeout              = time.Minute * 2
```

Comment:
just to keep it consistent can you make this 120 sec instead?

Reply:
ack